<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateStudentDialog</class>
 <widget class="QDialog" name="updateStudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Student</string>
  </property>
  <widget class="QLabel" name="updidnumberlabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>95</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ID NUMBER:</string>
   </property>
  </widget>
  <widget class="QLabel" name="updfirstnamelabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>184</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FIRST NAME:</string>
   </property>
  </widget>
  <widget class="QLabel" name="updlastnamelabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>249</y>
     <width>94</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LAST NAME:</string>
   </property>
  </widget>
  <widget class="QLabel" name="updyearlevellabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>313</y>
     <width>99</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>YEAR LEVEL:</string>
   </property>
  </widget>
  <widget class="QLabel" name="updgenderlabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>377</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GENDER:</string>
   </property>
  </widget>
  <widget class="QLabel" name="updprogramlabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>442</y>
     <width>84</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PROGRAM:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="updIDNumberLine">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="updfirstNameLine">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="updlastNameLine">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="updyearLevelBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="updgenderBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="updprogramBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>430</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="updatestudentLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>EDIT STUDENT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateStudentButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>520</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>UPDATE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
